src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -4,55 +4,34 @@ import { useUID } from 'react-uid';
 import styles from './FiltersDrawer.module.css';
 import Button from '../Button';
 import ButtonGroup from '../ButtonGroup';
-import Checkbox from '../Checkbox';
 import Drawer from '../Drawer';
-import FiltersCheckboxField from '../FiltersCheckboxField';
 import Heading from '../Heading';
 
 export type Props = {
   /**
-   * Checkbox fields that will be displayed in the filters drawer.
+   * Checkboxes or Checkbox fields that will be displayed in the filters drawer.
    */
-  checkboxFields: CheckboxField[];
-  /**
-   * Checked value keys mapped to their boolean checked status.
-   */
-  checkedMap: { [key: string]: boolean };
+  children: React.ReactNode;
   /**
    * CSS class names that can be appended to the component.
    */
   className?: string;
   /**
-   * Callback called when filters drawer is closed.
-   */
-  onClose: (checkedValues: { [key: string]: boolean }) => void;
-  /**
-   * Sets the filters drawer window open or closed.
-   */
-  isActive: boolean;
-};
-
-export type CheckboxField = {
-  /**
-   * Legend text string that names the fieldset.
+   * Callback called when Clear All button is called.
```

Comment:
Nit: I would avoid using exact button copy because those things shift around over time and you're probably not going to think about updating this comment (ex: "the clear button" instead of "Clear All button").

Reply:
Got it, will update